### DISCUSSION TITLE
Help with helper!

### DISCUSSION THREAD

**Discussion by k-g-j** [handlebars-lang/handlebars.js#1836] [Q&A] [answered]
I have this  div as a partial
``` html
  <div class='container post-image'>
    <img src='{{format_path post_image}}' class="w-50" >
  </div>
``` 
and this function in my helper folder 
``` js
  format_path: path => {
    return path.split('public').pop();
  }
}
```
the function basically returns "images/xyzxyz.jpeg" for my img src.
Everything works perfectly when I load the partial on my homepage, but when I load it from another view like my dashboard, I receive this message "TypeError: Cannot read properties of undefined (reading 'split')"
Both my dashboard and homepage are in the same "views" folder.

**Comment by jaylinski** [comment, marked as answer]
It seems like the `post_image`-variable is missing in your dashboard-view, making it `undefined`.

**Comment by k-g-j** [comment]
Yes that solved the problem! Thank you!!!!!
